recipes/overlay-files/files/overlay.sh

```diff
@@ -18,7 +18,7 @@ for file in $OVERLAY_FILES ; do
     if test -e "$overlay_file" ; then
 	if test ! -h "$link_file" -o \
 	"`readlink "$link_file"`" != "$overlay_file" ; then
-	    ln -sf "$overlay_file" "$link_file"
+	    ln -sfn "$overlay_file" "$link_file"
```

Comment:
Well, we need at the very least also -n. But I've also been confused by the wrong help output from busybox ln:

```
# ln -h
...
        -T      2nd arg must be a DIR
```
that should probably be "must not". But we also need -n, because unlike coreutils ln, busybox ln also takes -n into account when deciding whether the second argument is a directory. That is, in
```
touch a
mkdir d
ln -s d dl
ln -sfT a dl
```
the last command fails with busybox, but not coreutils... I'll update to include -T.